src/Domain/CoordinateMaps/Wedge.cpp

```diff
@@ -32,12 +37,16 @@ Wedge<Dim>::Wedge(const double radius_inner, const double radius_outer,
       radius_outer_(radius_outer),
       sphericity_inner_(sphericity_inner),
       sphericity_outer_(sphericity_outer),
+      cube_half_length_(cube_half_length),
+      focal_offset_(focal_offset),
       orientation_of_wedge_(std::move(orientation_of_wedge)),
       with_equiangular_map_(with_equiangular_map),
       halves_to_use_(halves_to_use),
       radial_distribution_(radial_distribution),
       opening_angles_(opening_angles) {
   const double sqrt_dim = sqrt(double{Dim});
+  const bool approx_zero_offset =
+      equal_within_roundoff(magnitude(focal_offset_), 0.0);
```

Comment:
Why not exactly zero offset?

Reply:
Good question! @AlexCarpenter46 @mar-celine Did we have a specific reason for checking the offset approximately == 0 instead of exactly == 0?

Reply:
Discussion on 8/21/24: Make this check for exactly zero. If it is, initialize the class exactly like if there was no offset with default arguments (e.g. for opening angles).